src/Microsoft.ManagedZlib/benchmarks/CompressFiles.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+using System.Text;
+
+namespace Microsoft.ManagedZLib.Benchmarks
```

Comment:
Consider converting to file-scoped namespaces in all your cs files. Saves you a bunch of tabs.

I think we can also make this a mandatory roslyn analyzer rule. Let me look for the rule ID and I'll give you the instructions.